compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/XCLBinGen.cpp

```diff
@@ -187,10 +187,20 @@ Expected<std::string> runTool(
     llvm::outs() << buffer.str();
   }
 
+  if (result != 0) {
+    return make_error<StringError>(
```

Comment:
might as well move this to right after the `sys::ExecuteAndWait`? doesn't really matter

Reply:
Yeah maybe printing whatever stdouts captured first will be useful though